### PR TITLE
set shortName as required property

### PR DESCRIPTION
**Pull request by ndoriomedov**

### CODE REVIEW

File: src/ProjectService.Data/ProjectRepository.cs [hunk 1]
```diff
@@ -216,9 +216,21 @@ public async Task<bool> DoesExistAsync(Guid projectId)
...
     }
 
+    public async Task<bool> DoesProjectNamesExistAsync(string name, string shortName)
+    {
+      return await _provider.Projects.AnyAsync(p =>
+        p.Name.ToLower().Equals(name.ToLower())
+        || p.ShortName.ToLower().Equals(shortName.ToLower()));
+    }
+
     public async Task<bool> DoesProjectNameExistAsync(string name)
```
Comment (MariyaSin) [inline src/ProjectService.Data/ProjectRepository.cs:226]: you in project repository, no need write the word "Project"

File: src/ProjectService.Validation/Project/CreateProjectRequestValidator.cs [hunk 2]
```diff
@@ -1,52 +1,38 @@
...
+      CascadeMode = CascadeMode.Stop;
 
       RuleFor(project => project.Name.Trim())
-        .Cascade(CascadeMode.Stop)
-        .MaximumLength(150).WithMessage("Project name is too long.")
-        .MustAsync(async (name, _) => !await projectRepository.DoesProjectNameExistAsync(name))
-        .WithMessage(project => $"Project with name '{project.Name}' already exists.");
+        .MaximumLength(150).WithMessage("Project name is too long.");
+
+      RuleFor(project => project.ShortName)
```
[the remark this answers is not in the document]
Comment (MariyaSin) [reply src/ProjectService.Validation/Project/CreateProjectRequestValidator.cs:26]: it makes sense if you check the minimum length

[the remark this answers is not in the document]
Comment (vorobevamv) [reply src/ProjectService.Validation/Project/CreateProjectRequestValidator.cs:26]: what if user if weird and there would be 20 letters and 30 spaces in name?)

[the remark this answers is not in the document]
Comment (ndoriomedov) [reply src/ProjectService.Validation/Project/CreateProjectRequestValidator.cs]: added trim)

**Review by vorobevamv — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**